### DISCUSSION TITLE
indepth analysis no colors for some langs since two weeks

### DISCUSSION THREAD

**Discussion by ttytm** [lowlighter/metrics#1310] [Q&A]
The workflow I'm using for metrics runs weekly to spit out some analytics.
Last week when the colors were not right for some languages, I thought it was just a small buggy exception. 
But since it's been occurring again for another week, I'll open a small issue here. There are no color problems with other analysis methods, only with in-depth.

(Edit: Python, Lua and Vim Script in this example) 

<img width=420 src="https://raw.githubusercontent.com/tobealive/tobealive/main/assets/langs-indepth-programming.svg">

**Comment by lowlighter** [comment]
Hi 👋 !

It could be a regression from #1118
I'll check it out, thanks for reporting 👍

**Reply by ttytm** [reply]
Thanks for the reply. Did you had a chance to check on it already?
